modular_skyrat/modules/altborgs/code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -1,3 +1,5 @@
+var/has_small_sprite = TRUE
```

Comment:
```suggestion
/obj/item/robot_model/var/has_small_sprite = TRUE
```

Reply:
I ended up reverting because I'm not sure how to call this variable in robot_upgrades.dm when it's defined in this manner